src/main/kotlin/nl/joelchrist/shorty/shorties/endpoints/ShortiesEndpoint.kt

```diff
@@ -53,6 +53,16 @@ class ShortiesEndpoint(@Autowired private val shortiesManager: ShortiesManager,
                     .body(UUID.randomUUID().let { ShortyMetadata(fileManager.create(it, file), it.toString()) })
                     .also { log.info("Creating file ${file.originalFilename}") }
 
+    @RequestMapping(value = ["/shorties/{identifier}"], method = [(RequestMethod.DELETE)])
```

Comment:
Waarom moet je shorties en files los deleten?

Reply:
Is voor later, als we ooit nog private shorties gaan delen. Zoals beschreven in dat nfc ticket van shorty. Dan wil je niet dat als die met iemand is gedeeld en hij heeft een shorty die verwijst naar die file dat die dan een 404 krijgt.